convert.ts

```diff
@@ -27,20 +27,75 @@ export const convertAsset = (
   asset: Pick<
     Asset,
     'id' | 'animationUrl' | 'image' | 'name' | 'unlockedContent'
-  >,
+  > & {
+    collection: Pick<Collection, 'address' | 'name'>
+    owned: {
+      aggregates: Maybe<{
+        sum: Maybe<Pick<OwnershipSumAggregates, 'quantity'>>
+      }>
+    }
+    bestBid: Maybe<{
+      nodes: Array<
+        Pick<Offer, 'unitPrice' | 'amount'> & {
+          currency: Pick<Currency, 'decimals' | 'symbol'>
+        }
+      >
+    }>
+  },
 ): {
   id: string
   animationUrl: string | null | undefined
   image: string
   name: string
+  collection: {
+    address: string
+    name: string
+  }
+  owned: BigNumber
   unlockedContent: { url: string; mimetype: string | null } | null
-} => ({
-  id: asset.id,
-  animationUrl: asset.animationUrl,
-  image: asset.image,
-  name: asset.name,
-  unlockedContent: asset.unlockedContent,
-})
+  bestBid:
+    | {
+        unitPrice: BigNumber
+        currency: {
+          decimals: number
+          symbol: string
+        }
+      }
+    | undefined
+} => {
+  const bestBid = asset.bestBid?.nodes[0]
+  if (!bestBid)
+    return {
+      id: asset.id,
+      animationUrl: asset.animationUrl,
+      image: asset.image,
+      name: asset.name,
+      collection: {
+        address: asset.collection.address,
+        name: asset.collection.name,
+      },
+      owned: BigNumber.from(asset.owned.aggregates?.sum?.quantity || '0'),
+      unlockedContent: asset.unlockedContent,
+      bestBid: undefined,
+    }
+
+  return {
+    id: asset.id,
+    animationUrl: asset.animationUrl,
+    image: asset.image,
+    name: asset.name,
+    collection: {
+      address: asset.collection.address,
+      name: asset.collection.name,
+    },
+    owned: BigNumber.from(asset.owned.aggregates?.sum?.quantity || '0'),
+    unlockedContent: asset.unlockedContent,
+    bestBid: {
+      unitPrice: BigNumber.from(bestBid.unitPrice),
+      currency: bestBid.currency,
+    },
+  }
```

Comment:
Instead of returning 2 type of object that can lead to confusion and mistakes
```suggestion
  return {
    id: asset.id,
    animationUrl: asset.animationUrl,
    image: asset.image,
    name: asset.name,
    collection: {
      address: asset.collection.address,
      name: asset.collection.name,
    },
    owned: BigNumber.from(asset.owned.aggregates?.sum?.quantity || '0'),
    unlockedContent: asset.unlockedContent,
    bestBid: bestBid ? {
      unitPrice: BigNumber.from(bestBid.unitPrice),
      currency: bestBid.currency,
    } : undefined,
  }
```